ovs/extensions/healthcheck/alba_hc.py

```diff
@@ -213,12 +214,11 @@ def check_if_proxies_work(result_handler):
                                 raise RuntimeError('Creation namespace has timed out after {0}s'.format(time.time() - namespace_start_time))
                             list_ns_osds_output = AlbaCLI.run(command='list-ns-osds', config=abm_config, extra_params=[namespace_key])
                             # Example output: [[0, [u'Active']], [3, [u'Active']]]
-                            namespace_ready = True
-                            for osd_info in list_ns_osds_output:  # If there are no osd_info records, uploading will fail so covered by HC
-                                osd_state = osd_info[1][0]
-                                if osd_state != 'Active':
-                                    namespace_ready = False
-                            if namespace_ready is True:
+                            num_active_osds = 0
```

Comment:
https://github.com/openvstorage/openvstorage-health-check/issues/378 stops me from accepting it :(

Reply:
You could instead leverage on the 'is_available' flag
So when the loop would not finish because of the output perform the following thing:
Find the alba backend in the Framework DAL
```
alba_backend = find backend code here
alba_backend.invalidate_dynamics('presets')
current_presets = alba_backend.presets
# Find the preset in the loop
preset = ...
if preset['is_available'] == True:
  # At this point, preset is satifiable
  break
```